src/pages/styles/plugins.scss

```diff
@@ -1,5 +0,0 @@
-.dark .plugins-page-wrapper {
-    .ant-tabs-tab:not(.ant-tabs-tab-active) {
-        color: white;
-    }
-}
```

Comment:
Looks good to me (I made some last copy changes, take a look at that), but seems like this piece of CSS is stil needed – see James's comment.